crates/nu-explore/src/nu_common/command.rs

```diff
@@ -88,7 +90,8 @@ fn eval_source2(
     //
     // So we LITERALLY ignore all expressions except the LAST.
     if block.len() > 1 {
-        block.pipelines.drain(..block.pipelines.len() - 1);
+        let range = ..block.pipelines.len() - 1;
+        Arc::make_mut(&mut block).pipelines.drain(range);
```

Comment:
Can this be written on a single line like this?

In case `strong_count > 1` we would get the copy in the return value and mutate there. 

(OK I checked https://doc.rust-lang.org/src/alloc/sync.rs.html#2131 will in fact reassign the in `&mut` to the created copy)

Reply:
probably not on a single line, but the `make_mut` can be relocated to be on the other line instead